SpatialGDK/Documentation/content/tutorials/tutorial-porting-guide.md

```diff
@@ -1,41 +1,45 @@
-
-* If you haven't done this already, follow our _Get started_ guide before porting your game:
-    * [Get started: 1 - Dependencies]({{urlRoot}}/content/get-started/dependencies)
-    * [Get started: 2 - Get and build the GDK’s Unreal Engine Fork]({{urlRoot}}/content/get-started/build-unreal-fork)
-    * Get started: 3 - Set up a project</br> 
-    We recommend the [Example Project]({{urlRoot}}/content/get-started/example-project/exampleproject-intro)
-</br>
-</br>
-* Open a terminal window and run the command `spatial update` to ensure your [spatial CLI]({{urlRoot}}/content/glossary#spatialos-command-line-tool-cli) installation is up to date. 
+**Tip: Reference project** 
+<br/>
+ As you port your own Unreal project to SpatialOS, you could use our pre-ported [Unreal Shooter Game](https://docs.unrealengine.com/en-us/Resources/SampleGames/ShooterGame) as a reference. You should already have this project as it is included in the `Samples` directory of [the SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine) which you downloaded as part of the _Get Started_ steps. </br>
+(If you want to see the game running, there's a [video on youtube](https://www.youtube.com/watch?v=xojgH7hJgQs&feature=youtu.be) to check out.) 
 
 **Terms used in this guide**</br>
 * `<GameRoot>` - The directory containing your project's `.uproject` file and `Source` directory.  
 * `<ProjectRoot>` - The directory containing your `<GameRoot>`.  
 * `<YourProject>` - The name of your project's `.uproject` file (for example, `\<GameRoot>\TP_SpatialGDK.uproject`).
 
-<!-- <%(Callout type="tip" message=" -->
-<!-- ")%> -->
+</br>
+## 1. Before you start
 
-> **TIP: Reference project** 
-<br /><br />
-> As you port your own Unreal project to SpatialOS, you could use our pre-ported [Unreal Shooter Game](https://docs.unrealengine.com/en-us/Resources/SampleGames/ShooterGame) as a reference. You should already have this project as it is included in the `Samples` directory of [the SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine) which you downloaded as part of the _Get Started_ steps. 
-<br /><br />
-> (If you want to see the game running, there's a [video on youtube](https://www.youtube.com/watch?v=xojgH7hJgQs&feature=youtu.be) to check out.) 
+Before porting your project: 
 
+* Set up the GDK and Starter Template.</br>
+If you haven't done this already, follow the three steps in _Get started_ guide in order to port your game:
+    1. [Get started 1 - Dependencies]({{urlRoot}}/content/get-started/dependencies)
+    1. [Get started 2 - Get and build the GDK’s Unreal Engine Fork]({{urlRoot}}/content/get-started/build-unreal-fork)
+    1. [Get started 3 - Set up a project: The Starter Template]({{urlRoot}}/content/get-started/gdk-template) </br>
+    (Note that you must follow the Starter Template instructions and not the Example Project instructions.)
+</br>
+</br>
+* Update your local version of the spatial CLI. s</br>
```

Comment:
```suggestion
* Update your local version of the SpatialOS CLI. s</br>
```
Also I have no idea what the "s" is doing here?? But it doesn't seem to render on docs-testing.